materialsLibrary/src/simple/simple.fragment.fx

```diff
@@ -13,6 +13,8 @@ varying vec3 vNormalW;
 
 #ifdef VERTEXCOLOR
 varying vec4 vColor;
+#elif INSTANCESCOLOR

```

Comment:
can we use #if defined(VERTEXCOLOR) || defined(INSTANCESCOLOR) to avoid code duplication ?